devops/jobs/ClusterInstanceMonitoring.groovy

```diff
@@ -0,0 +1,114 @@
+                
+                extraVars['CONFIGURATION_INTERNAL_REPO'] = config_internal_repo
+
+                parameters{
+                    stringParam('CONFIGURATION_REPO', extraVars.get('CONFIGURATION_REPO', 'https://github.com/edx/configuration.git'),
+                                    'Git repo containing edX configuration.')
+                    stringParam('CONFIGURATION_BRANCH', extraVars.get('CONFIGURATION_BRANCH', 'hchen/cluster-monitoring'),
+                            'e.g. tagname or origin/branchname')
+
+                    stringParam('CONFIGURATION_INTERNAL_REPO', extraVars.get('CONFIGURATION_INTERNAL_REPO',  "git@github.com:edx/${deployment}-internal.git"),
+                    		'Git repo containing internal overrides')
+            		stringParam('CONFIGURATION_INTERNAL_BRANCH', extraVars.get('CONFIGURATION_INTERNAL_BRANCH', 'master'),
+                    		'e.g. tagname or origin/branchname')
+                }
+                
+                multiscm{
+                    git {
+                        remote {
+                            url('$CONFIGURATION_REPO')
+                            branch('$CONFIGURATION_BRANCH')
+                        }
+                        extensions {
+                            cleanAfterCheckout()
+                            pruneBranches()
+                            relativeTargetDirectory('configuration')
+                        }
+                    } 
+
+                    git {
+                        remote {
+                            url('$CONFIGURATION_INTERNAL_REPO')
+                            branch('$CONFIGURATION_INTERNAL_BRANCH')
+                                if (gitCredentialId) {
+                                    credentials(gitCredentialId)
+                                }
+                        }
+                        extensions {
+                            cleanAfterCheckout()
+                            pruneBranches()
+                            relativeTargetDirectory('configuration-internal')
+                        }
+                    }
+                }
+
+                throttleConcurrentBuilds {
+                    maxPerNode(0)
+                    maxTotal(0)
+                }
+
+                triggers {
+                    cron("H/10 * * * *")
+                }
+
+                environmentVariables {
+                    env('REGION', extraVars.get('REGION','us-east-1'))
+                }
+
+                steps {
+                    virtualenv {
+                        nature("shell")
+                        systemSitePackages(false)
+
+                        command(
+                            dslFactory.readFileFromWorkspace("devops/resources/cluster-instance-monitoring.sh")
+                        )
+
+                    }
+
+                }
+
+                publishers {
+                    mailer(extraVars.get('NOTIFY_ON_FAILURE','devops@edx.org'), false, false)
```

Comment:
Don't hardcode us into a public repo, better to conditionally set publishers based on NOTIFY_ON_FAILURE being empty or not.